### PR TITLE
Add Subhoofer and Scrollscope

### PR DESCRIPTION
**Pull request by ardura**

### CODE REVIEW

File: src/plugins/ardura/Scrollscope/1.4.0.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+name: Scrollscope
+author: Ardura
+homepage: https://github.com/ardura/Scrollscope
+description: A simple scrolling oscilloscope in rust with multichannel inputs and an analyzer
+date: 2024-07-12T07:00:00.000Z
+license: GPL3
```
Comment (kmturley) [inline src/plugins/ardura/Scrollscope/1.4.0.yaml]: Can you update to a value that matches GitHub licenses?
`license: gpl-3.0`

File: src/plugins/ardura/Scrollscope/1.4.0.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+tags:
+  - Effect
```
Comment (kmturley) [inline src/plugins/ardura/Scrollscope/1.4.0.yaml:9]: Any other sub categories which would help categorize it?

File: src/plugins/ardura/Subhoofer/2.2.1.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+name: Scrollscope
+author: Ardura
+homepage: https://github.com/ardura/Scrollscope
+description: A simple scrolling oscilloscope in rust with multichannel inputs and an analyzer
+date: 2024-07-12T07:00:00.000Z
+license: GPL3
```
Comment (kmturley) [inline src/plugins/ardura/Subhoofer/2.2.1.yaml]: Can you update to a value that matches GitHub licenses?
`license: gpl-3.0`

File: src/plugins/ardura/Subhoofer/2.2.1.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+tags:
+  - Effect
```
Comment (kmturley) [inline src/plugins/ardura/Subhoofer/2.2.1.yaml:9]: Same comment for more tags

**Review by kmturley — CHANGES_REQUESTED** [review]

File: src/plugins/ardura/Scrollscope/1.4.0.yaml (hunk 1 above)
Comment (ardura) [reply src/plugins/ardura/Scrollscope/1.4.0.yaml]: updated in latest commit

File: src/plugins/ardura/Subhoofer/2.2.1.yaml (hunk 3 above)
Comment (ardura) [reply src/plugins/ardura/Subhoofer/2.2.1.yaml]: updated in latest commit

File: src/plugins/ardura/Scrollscope/1.4.0.yaml (hunk 2 above)
Comment (ardura) [reply src/plugins/ardura/Scrollscope/1.4.0.yaml:9]: This plugin is for analysis so unfortunately that's the only category it fits under right now

File: src/plugins/ardura/Subhoofer/2.2.1.yaml (hunk 4 above)
Comment (ardura) [reply src/plugins/ardura/Subhoofer/2.2.1.yaml:9]: I added distortion because technically it adds some saturation in its effect

**Pull request merged**